scripts/webpack/sass.rule.js

```diff
@@ -23,8 +23,14 @@ module.exports = function(options) {
             config: { path: __dirname + '/postcss.config.js' }
           }
         },
-        { loader:'sass-loader', options: { sourceMap: options.sourceMap } }
+        { loader: 'sass-loader', options: { sourceMap: options.sourceMap } }
       ],
+      fallback: [{
+        loader: 'style-loader',
```

Comment:
I think we can just use the ExtractTextPlugin (and skip the extractSass bool logic) and just add a fallback option to the ExtractTextPlugin